acceptance/tests/direct_puppet/static_catalog_env_control.rb

```diff
@@ -0,0 +1,348 @@
+test_name "Environment control of static catalogs" 
+require 'json'
```

Comment:
I wasn't sure this was guaranteed to work, as `json` is not listed in our [acceptance Gemfile](https://github.com/puppetlabs/puppet/blob/stable/acceptance/Gemfile). But it looks like we `require 'json'` in at least one [other acceptance test](https://github.com/puppetlabs/puppet/blob/stable/acceptance/tests/modules/install/nonexistent_module.rb#L31-L33), so I guess it's ok.